src/common/__tests__/useMountId-test.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import useMountId from '../useMountId';
+
+type Props = {
+    onMount: (uuid: string) => void;
+};
+
+describe('useMountId', () => {
+    function TestComponent({ onMount }: Props): JSX.Element {
+        const uuid = useMountId(onMount);
+
+        return <div data-testid="uuid" data-uuid={uuid} />;
+    }
+
+    const getWrapper = (onMount: (uuid: string) => void): ReactWrapper => mount(<TestComponent onMount={onMount} />);
+
+    test('should render the component with a generated uuid and call the callback with the generated uuid', () => {
+        const onMount = jest.fn();
+        const wrapper = getWrapper(onMount);
+
+        expect(wrapper.find('[data-testid="uuid"]').prop('data-uuid')).toEqual(expect.any(String));
```

Comment:
Should we mock the underlying uuid library so we can assert that the string is non-empty, as well?